terraform.tfvars.example

```diff
@@ -0,0 +1,10 @@
+# metal_project_id="uuid"
```

Comment:
README.md must be updated:
- the renaming of these varaibles to metal_ prefix
- the availability of `metal env` as a way to populate these values
- the availability of terraform.tfvars.example as a starting point